cfme/configure/access_control.py

```diff
@@ -1256,20 +1266,73 @@ def is_displayed(self):
         )
 
 
-class EditTenantView(TenantForm):
+class EditTenantView(View):
     """ Edit Tenant View """
+    form = View.nested(TenantForm)
     save_button = Button('Save')
     reset_button = Button('Reset')
 
     @property
     def is_displayed(self):
         return (
-            self.accordions.accesscontrol.is_opened and
-            self.title.text == 'Editing Tenant "{}"'.format(self.context['object'].name)
+            self.form.accordions.accesscontrol.is_opened and
+            any([self.form.title.text == 'Editing Tenant "{}"'.format(self.context['object'].name),
+                self.form.title.text == 'Editing Project "{}"'.format(self.context['object'].name)])
         )
+        # used any() because `project` inherits `tenant` and edit page can have any one in name
+
+
+class TenantCollection(BaseCollection):
+    """Collection class for Tenant"""
+
+    def __init__(self, appliance):
+        self.appliance = appliance
+
+    def instantiate(self, name, description=None, parent=None, default=False):
+        return Tenant(collection=self, name=name, description=description, parent_tenant=parent,
+                      _default=default)
+
+    def get_root_tenant(self):
+        return self.instantiate('My Company', default=True)
+
+    def create(self, name, description, parent):
+
+        if isinstance(parent, Project):
+            raise ValueError("Projects cannot have children.")
+
+        tenant = self.instantiate(name, description, parent)
 
+        view = navigate_to(tenant.parent_tenant, 'Details')
+        view.toolbar.configuration.item_select('Add child Tenant to this Tenant')
 
-class Tenant(Updateable, Pretty, Navigatable):
+        view = self.create_view(AddTenantView)
+        changed = view.form.fill({'name': name,
+                   'description': description})
+        if changed:
+            view.form.add_button.click()
+        else:
+            view.form.cancel_button.click()
+
+        view = self.create_view(ParentDetailsTenantView)
+        view.flash.assert_success_message('Tenant "{}" was saved'.format(name))
+
+        return tenant
+
+    def delete(self, *tenants):
+        tenants = list(tenants)
```

Comment:
**REQUIRED**: Because you don't mutate `tenants`, this line is not needed.

Reply:
removed.